unison-src/transcripts/isTest-exists.output.md

```diff
@@ -0,0 +1,33 @@
+This transcript tests that UCM can always access the definition of
+`IsTest`/`isTest`, which is used internally.
+
+```ucm
+.> builtins.merge
+
+  Done.
+
+```
+```unison
+test> pass = [Ok "Passed"]
+```
+
+```ucm
+.> add
+
+  ⍟ I've added these definitions:
+  
+    pass : [Result]
+
+.> links pass
+
+  1. builtin.metadata.isTest : IsTest
+  
+  Tip: Try using `display 1` to display the first result or
+       `view 1` to view its source.
+
+.> display 1
+
+  IsTest
+
+```
+The definition and type of `isTest should exist.
```

Comment:
Unrelated: @pchiusano I'm not sure why this `` `isTest `` isn't terminated; it is terminated in the input markdown.

Reply:
I have noticed this bug. It's bizarre and only comes up for some transcripts, so good that we have this as an example. I'll file a separate ticket for that.

Reply:
added #1809